server/src/data/models.rs

```diff
@@ -2448,7 +2448,7 @@ pub struct DatasetEventCount {
     "server_configuration": {
         "LLM_BASE_URL": "https://api.openai.com/v1",
         "EMBEDDING_BASE_URL": "https://api.openai.com/v1",
```

Comment:
I think you missed `EMBEDDING_BASE_URL` in a few places and also didn't change the `DIMENSION` whatever that is called